test-data/rtrtr.conf

```diff
@@ -85,6 +86,7 @@ listen = [ "127.0.0.1:9326" ]
 unit = "rtr-tls-loop"
 certificate = "localhost.crt"
 key = "localhost.key"
+history-size = 12
```

Comment:
Is there a test somewhere that verifies that this non-default `history-size` is honoured?

Reply:
No, unfortunately not. We don’t have any run-time testing as of now. That should probably be fixed.